M/Monads/Versions.toml

```diff
@@ -1,2 +1,5 @@
 ["0.1.0"]
 git-tree-sha1 = "57af3fb7c17aaf677bd36dd68dc5048f33ef09db"
+
+["0.2.3"]
```

Comment:
0.2.0?

Reply:
Well, I spent 0.2.0…0.2.3 trying to figure out how to use Registrator. 0.2.3 was the first that worked. 0.2.0 is already on github with some previous commit. So I'd bet for 0.2.3. But if you think we could still do 0.2.0, I'd be fine with that. What should I do?